app/src/main/java/com/example/mycleaninglog/dto/cleaningTask.kt

```diff
@@ -2,9 +2,9 @@ package com.example.mycleaninglog.dto
 
 import com.google.firebase.firestore.Exclude
 
-data class cleaningTask(var cleaningTaskName:String = "", var cleaningTaskId:String = ""){
+data class Task(var taskName:String = "", var taskId:String = ""){
 
     override fun toString(): String {
-        return "$cleaningTaskName"
+        return "$taskName"
```

Comment:
I agree... Task is a more flexible name, and is PascalCase, as it should be (not lowerCamelCase).